Makefile

```diff
@@ -20,7 +20,7 @@
 CPUS      := $(shell getconf _NPROCESSORS_ONLN 2>/dev/null || getconf NPROCESSORS_ONLN 2>/dev/null || echo 1)
 
 PREFIX    = "/usr/local"
-VERSION   = `cmake -P config.cmake | grep -oP "VERSION\s+\K.*"`
+VERSION   = `cmake -P config.cmake | sed -n -e "s/^.*VERSION  *//p"`
```

Comment:
Doesn't matter too much here, but generally it's better to use `:=` than `=` for assignment and `$(shell <cmd>)` rather than backticks. See https://stackoverflow.com/a/1435867

Reply:
Works with GNU make only though. Not sure we can take that for granted everywhere?

Reply:
Let's see...

Reply:
It's already used to set `CPUS` a few lines above and nobody has complained yet.

Reply:
Sure, for Linux. For Windows/MinGW too, not so sure about Mac though?